PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java

```diff
@@ -188,22 +164,21 @@ protected JComponent createCenterPanel() {
         return AzureTaskManager.getInstance().runInModalAsObservable(task).toSingle();
     }
 
+    private static final IdeaSecureStore secureStore = IdeaSecureStore.getInstance();
```

Comment:
move to header

Reply:
resolved in https://github.com/microsoft/azure-tools-for-java/pull/5265